### PR TITLE
Add

### PR DESCRIPTION
**Pull request by Smelky**

### CODE REVIEW

File: src/main/java/Comment.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import com.google.api.client.util.DateTime;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Comment {
```
Comment (Anderwerty) [inline src/main/java/Comment.java]: It is better to use a builder here. You can create it using  Lombok annotation.

File: src/main/java/YoutubeCommentParser.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    private static YouTube youtube;
+    private List<YoutubeVideo> listOfVideos = Lists.newArrayList();
+    private static List<String> scopes = Lists.newArrayList("https://www.googleapis.com/auth/youtube.force-ssl");
+
+    public void init() throws Exception {
+        Credential credential = null;
+        try {
+            credential = Auth.authorize(scopes, "commentThreads");
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (Anderwerty) [inline src/main/java/YoutubeCommentParser.java]: Why did you use logger here?

File: src/main/java/YoutubeCommentParser.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        }
+    }
+
+    private static List<CommentThread> getCommentThreads(String videoId) {
+        CommentThreadListResponse videoCommentsListResponse = null;
+        try {
+            videoCommentsListResponse = youtube.commentThreads().list("snippet")
+                    .setVideoId(videoId).setTextFormat("plainText").execute();
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (Anderwerty) [inline src/main/java/YoutubeCommentParser.java]: The same

File: src/main/java/YoutubeCommentParser.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        }
+        return videoCommentsListResponse.getItems();
```
Comment (Anderwerty) [inline src/main/java/YoutubeCommentParser.java]: You will get NPE here if you get IOException.

File: src/main/resources/client_secrets.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (Anderwerty) [inline src/main/resources/client_secrets.json]: Better not to publish credential into the public repository.  Just add it into gitignore and remove from repo.

Comment (Smelky) [reply src/main/resources/client_secrets.json]: did it pls review

File: .gitignore [hunk 6]
```diff
@@ -1,3 +1,4 @@
 .idea
 *.iml
-target
\ No newline at end of file
+target
+*.json
```
Comment (Anderwerty) [inline .gitignore]: Please add a blank line at the end of the file.

Comment (Smelky) [reply .gitignore]: hmm, i did it, but on github i don`t see the blank line in gitignore file

**Pull request merged**